locales/en.yaml

```
@@ -21,34 +21,35 @@ system:
 commands:
   groups:
     summarization:
-      name: 量子速读
+      name: Quantum Speed Reading
```

```suggestion
      name: Quantum Speed-Reading
```

Should we align the name for it?